### PR TITLE
Backend folder

### PR DESCRIPTION
**Pull request by SCavus**

### CODE REVIEW

**Comment by SCavus** [comment]
> @SCavus Please use express generator - https://expressjs.com/en/starter/generator.html, it will generate all the boilerplate code needed to setup express.

I have now installed express-generator and should be ok now

**Comment by dahfool** [comment]
> > @SCavus Please use express generator - https://expressjs.com/en/starter/generator.html, it will generate all the boilerplate code needed to setup express.
> 
> I have now installed express-generator and should be ok now

👍

File: backend/app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.use('/', indexRouter);
+app.use('/users', usersRouter);
+
+// catch 404 and forward to error handler
+app.use(function(req, res, next) {
```
Comment (AlexeyBrambalov) [inline backend/app.js:26]: I have no idea what is "next" doing, and 'jade". Are you sure about this starter file?

Comment (SCavus) [reply backend/app.js:26]: Me neither actually. I have used express-generator which automatically creates necessary files. It was a recommendation from Gerald.

**Pull request merged**